recipes/wsidata/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "wsidata" %}
+{% set version = "0.7.0" %}
+{% set python_min = "3.10" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/wsidata-{{ version }}.tar.gz
+  sha256: 0b327c716fb3abcf4fe5ef4d3873eb9259436e6fedf52af23c373165639dd87a
+  patches:
+    - openslide.patch
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python {{ python_min }}
+    - hatchling
+    - pip
+  run:
+    - python >={{ python_min }}
+    - spatialdata >=0.3.0
+    - opencv
+    - openslide-python >=1.4.1
```

Comment:
This package does not exist for windows, you will have to skip it for here if you want to pass, or make a PR there to get it to move forward:

https://github.com/conda-forge/openslide-python-feedstock/blob/main/recipe/meta.yaml#L14

Reply:
OpenSlide Python works fine on Windows, but depends on OpenSlide.  OpenSlide builds fine on MinGW, and builds with clang-cl after the next release or with [this patch](https://github.com/microsoft/vcpkg/blob/master/ports/openslide/fix-win-build.patch) (which assumes `-Dtest=disabled`).  OpenSlide does not build with MSVC and probably never will.

Reply:
You found the tracking issue. PRs welcome!

Reply:
Great @bgilbert, thank you! 

I will wait for your PR. Until then, I am marking the skip for windows.